tools/pd-ctl/pdctl/command/plugin_command.go

```diff
@@ -122,7 +92,17 @@ func unloadPluginCommandFunc(cmd *cobra.Command, args []string) {
 		cmd.Println(err)
 		return
 	}
-	_, err = doRequest(cmd, pluginPrefix, http.MethodDelete, WithBody("application/json", bytes.NewBuffer(reqData)))
+	switch action {
+	case "load":
```

Comment:
would you like to avoid use magic string in code? It will be better to be defined const.